cli/scan/scan.go

```diff
@@ -42,17 +46,20 @@ func scanMain(args []string, c cli.Config) (err error) {
 				return
 			}
 
-			fmt.Printf("Scanning %s...\n", host)
-
-			var results map[string]scan.FamilyResult
-			results, err = scan.Default.RunScans(host, c.IP, c.Family, c.Scanner, c.Timeout)
-			if err != nil {
-				return
-			}
-			if results != nil {
-				printJSON(results)
-			}
+			wg.Add(1)
+			go func(host string) {
```

Comment:
Cloud make it a function instead of using the context from the closure. That way it is clear what it depends on and only things it needs are passed on.